src/test/modelconfig_test.cpp

```diff
@@ -439,3 +439,32 @@ TEST(ModelConfig, modelVersionPolicyIncorrect) {
     auto result = config.parseModelVersionPolicy(command);
     EXPECT_EQ(result, ovms::StatusCode::MODEL_VERSION_POLICY_UNSUPPORTED_KEY);
 }
+
+TEST(ModelConfig, ConfigParseNodeWithForbiddenShapeName) {
+    const char* config = R"#({
+        "model_config_list": [
+            {
+                "config": {
+                    "name": "alpha",
+                    "base_path": "/tmp/models/dummy1",
+                    "shape": {"ANONYMOUS_INPUT_NAME": "(1, 3, 600, 600)"}
```

Comment:
Can we somehow avoid hardcoding ANONYMOUS_INPUT_NAME? We already have it defined somewhere else under alias `ANONYMOUS_INPUT_NAME`. Maybe we could somehow use it here?